src/test/data/XmlSerializableConciergeTest/typicalConcierge.xml

```diff
@@ -50,8 +50,8 @@
                 <tagged>vip</tagged>
             </guest>
             <bookingPeriod>
-                <startDate>24/10/2018</startDate>
-                <endDate>30/10/2018</endDate>
+                <startDate>25/10/2018</startDate>
```

Comment:
I see that these have been added to the .gitignore file.

Are the dates generated based on the current (today)'s date? Maybe it is not needed to commit these.

Reply:
Yes the dates are generated based on the day's date. It is crucial to do so because our tests are dependent on dates, so hard-coding the dates would make them fail on certain days of the year. Thus, the solution is to keep the dates dynamic and since I made the effort to automate the creation of dynamic XML test files, the solution is to add these dynamically changing files to the .gitignore.